core/file/ctime_spec.rb

```diff
@@ -3,15 +3,23 @@
 describe "File.ctime" do
   before :each do
     @file = __FILE__
+    3.times do
+      @ctime = File.ctime(@file)
+      break if @ctime.usec > 0
+      sleep 0.1
+    end
```

Comment:
ctime will not change just by retrying, creating another file with `tmp()` would be more helpful.